raiden_contracts/data/source/services/UserDeposit.sol

```diff
@@ -6,9 +6,9 @@ import "raiden/Token.sol";
 import "raiden/Utils.sol";
 
 contract UserDeposit is Utils {
-    uint constant public withdraw_timeout = 15 * 100;  // time before withdraw is allowed in seconds
+    uint public withdraw_timeout;  // time before withdraw is allowed in seconds
 
-    // Token to be used for the deposit
+    // Token to be used foruser_deposit_withdraw_timeout the deposit
```

Comment:
nitpick: small typo, missing space in `for user_deposit...`

Reply:
🙈

Reply:
Done ✔️ 
As this is a single commit, I did amend it instead of a fixup commit.